### PR TITLE
bug/feedback-spacing-issues

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?
https://trello.com/c/VgnFAdVB/848-feedback-display-issues

> If this is an issue, do we have steps to reproduce?
Display of feedback (padding etc.) varies between pages. Display issues on some white text on black pages.

### Intent

> What changes are introduced by this PR that correspond to the above card?
Update all pages to use a common parent template. 
Standardise certain page elements:
- navigation bars
- titles
- content positioning
- feedback
Sub template for media pages; audio and video.

> Would this PR benefit from screenshots?
It would benefit from page to page testing, runs through.

### Considerations

> Is there any additional information that would help when reviewing this PR?
All the pages will have changed slightly. This should be tested by the content and design teams.

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/routes/tags.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const createTagRouter = ({ cmsService }) => {
       );
 
       const pageType = ['tags', 'series'].includes(data.contentType)
-        ? 'tags'
+        ? 'tagsTopicsSeries'
```
Comment (jonhindle) [inline server/routes/tags.js:41]: Renamed to highlight actual use case.

File: assets/sass/components/_topics-list.scss [hunk 2]
```diff
@@ -1,29 +0,0 @@
-.topics-list {
```
Comment (jonhindle) [inline assets/sass/components/_topics-list.scss:1]: Tidying up - File removed as I think it's the old topics list from the media pages.

File: server/views/pages/search.html [hunk 3]
```diff
@@ -1,60 +1,26 @@
...
+    {% if results[0] %}
+      <h3 class="govuk-heading-m">Showing results for "{{ query }}"</h3>
+      <ul class="govuk-list">
+        {% for item in results %}
+          <li>
+            <a class="govuk-link" href="{{item.url}}">
+              <span class="govuk-!-font-weight-bold">{{ item.title }}</span>
+              <p class="govuk-body">
+                {% if item.summary %}{{ item.summary }}
+                {% else %}No summary available{% endif%}
```
Comment (BarryGeeMOJ) [inline server/views/pages/search.html:14]: Is it not cleaner to perform checks like this (`if item.summary....`) in the controller and just pass the result through to the view? Would make cases like this more easily testable and cleans up the view a little?

File: server/views/pages/games/chess.html [hunk 4]
```diff
@@ -1,146 +1,120 @@
...
-              <div class="govuk-details__text">
-              Roughly equivalent to "optimism." Positive values of contempt favor more "risky" play, while negative values will favor draws. Zero is neutral.
-            </div>
-            </details>
-          </div> #}
-            <div class="govuk-form-group">
+          {# <div class="govuk-form-group"> #}
+          {# <label for="timeInc" class="govuk-label">Increment (sec)</label> #}
+          <input type="hidden"  id="timeInc" value="2"/>
+          {# </div> #}
```
Comment (BarryGeeMOJ) [inline server/views/pages/games/chess.html:58]: Can the commented code be removed if it's not needed?

File: server/views/pages/games/chess.html [hunk 5]
```diff
@@ -1,146 +1,120 @@
...
-    <div class="govuk-main-wrapper">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-          <h1 id="title" class="govuk-heading-xl govuk-!-margin-bottom-4 govuk-!-margin-top-3">Chess</h1>
+{% block content %}
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <div class="game-board-container">
+          <span class="govuk-heading-s govuk-!-margin-bottom-2 govuk-hub-computer" id="time1"></span>
```
Comment (davidthomas22) [inline server/views/pages/games/chess.html:18]: govuk-hub-computer & govuk-hub-computer are being overidden by govuk-heading-s

File: server/views/pages/npr.html [hunk 6]
```diff
@@ -55,16 +44,6 @@ <h1 id="title" class="govuk-hub-media-player__title govuk-heading-xl govuk-body
     >
       <source src="{{ data.media }}" type="audio/ogg"/>
     </audio>
```
Comment (davidthomas22) [inline server/views/pages/npr.html:46]: `govuk-hub-on-black-feedback` class not imported  as it is not using the `mediaTemplate.njk`

File: server/views/pages/tagsTopicsSeries.html [hunk 7]
```diff
@@ -47,26 +36,6 @@ <h1 id="title" class="govuk-heading-xl">{{ title }}</h1>
     data: data.relatedContent,
     lazyLoadContent: true
   }) }}
-
-
-  {% if not data.excludeFeedback %}
-    <div class="govuk-form-group govuk-hub-article-feedback govuk-hub-on-black-feedback">
```
Comment (davidthomas22) [inline server/views/pages/tagsTopicsSeries.html:53]: `govuk-hub-article-feedback` handled the border on this page type. Requires a `<hr>` as applied in the `tagsCategories.html`

File: assets/sass/_general.scss [hunk 8]
```diff
@@ -74,13 +70,10 @@
 }
 
 .govuk-hub-comment-form__footer {
-  text-align: right;
   @include govuk-font($size: 19);
-}
-
-.govuk-hub-comment-form__footer::after,
-.govuk-hub-feedback-padding {
```
Comment (davidthomas22) [inline assets/sass/_general.scss:82]: Removing seems to have reduced the padding on the feedback widget. Class still called in `server/views/components/feedback-widget/template.njk`

File: server/views/pages/games/anagramica.html [hunk 9]
```diff
@@ -1,194 +1,168 @@
...
-
-{% block pageTitle %}
-  {{ title }}
-{% endblock %}
+{% extends "../../components/basicTemplate.njk" %}
 
 {% block head %}
 	<link href="/public/javascript/games/anagramica/stylesheets/style.css" rel="stylesheet"/>
-  <link href="/public/stylesheets/application.css" rel="stylesheet"/>
+  {{ super() }}
```
Comment (davidthomas22) [inline server/views/pages/games/anagramica.html:7]: Sorry, not sure where `super()` was called and what it does

File: server/routes/approvedVisitors.js [hunk 10]
```diff
@@ -42,6 +42,7 @@ const createApprovedVisitorsRouter = () => {
   //         detailsType: 'small',
   //         returnUrl,
   //         ...personalisation,
+  //         data: { contentType: 'profile' },
   //       });
   //     } catch (e) {
   //       return next(e);
```
[the remark this answers is not in the document]
Comment (jonhindle) [reply server/routes/approvedVisitors.js:48]: So... approved visitors is a page that should/could be being reinstated at some point.

File: server/views/pages/games/anagramica.html (hunk 9 above)
Comment (jonhindle) [reply server/views/pages/games/anagramica.html:7]: https://mozilla.github.io/nunjucks/templating.html#super
pulls in the parent blocks contents

File: server/views/pages/error.html [hunk 11]
```diff
@@ -1,36 +1,25 @@
...
+{% set title = 'The error has been logged, please try again' %}
+{% set longTitle = 'Something went wrong' %}
 
-{% block head %}
-  <link href="/public/stylesheets/application.css" rel="stylesheet"/>
+{% block content %}
+  <p class="govuk-body">
+    Return to the <a href="/">home page</a> or
+    <script>
+      document.write('<a href="' + document.referrer + '">try again</a>');
```
[the remark this answers is not in the document]
Comment (jonhindle) [reply server/views/pages/error.html:10]: We could just put the url in server side via nunjucks?

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Review by BarryGeeMOJ — APPROVED** [review]

**Pull request merged**